scripts/ci/github-preview-link.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+PREVIEW_URL="https://dweb.link$CONTENT_PATH"
+API_PARAMS=$(jq --monochrome-output --null-input \
+    --arg state "success" \
+    --arg target_url "$PREVIEW_URL" \
+    --arg description "Preview updated website on IPFS" \
+    --arg context "Preview is ready" \
+    '{ state: $state, target_url: $target_url, description: $description, context: $context }' )
+curl --output /dev/null --silent --show-error \
+    -X POST -H "Authorization: Bearer $GITHUB_TOKEN" -H 'Content-Type: application/json' \
+    --data "$API_PARAMS" 'https://api.github.com/repos/ipfs/distributions/statuses/${GIT_REVISION}'
```

Comment:
:information_source: we could replace this script with JS call and  [`actions/github-script`](https://github.com/actions/github-script) but I don't want to sink any more time into refactoring this PR ;)